code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -231,10 +231,15 @@
 			var/amount_per_pill = reagents.total_volume/count
 			if (amount_per_pill > 60) amount_per_pill = 60
 
-			var/name = sanitizeSafe(input(usr,"Name:","Name your pill!","[reagents.get_master_reagent_name()] ([amount_per_pill] units)") as null|text, MAX_NAME_LEN)
+			if(condi)//For the condimaster
+				var/name = sanitizeSafe(input(usr,"Name:","Name your cube!","[reagents.get_master_reagent_name()] ([amount_per_pill] units)") as null|text, MAX_NAME_LEN)

```

Comment:
This only exists within the scope of this if statement. The variable declaration needs to be moved outside of it